### PR TITLE
Add components and style

### PR DESCRIPTION
**Pull request by karam084**
**Description**
Continue with the development of the Math Magicians app. You will develop a React component.

**Project requirements**

- [ ] Delete all the boilerplate from CRA (text, images, styles).
- [ ] Create a directory called components.
- [ ] Inside components create a new Calculator.js file.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (FrederickMih) [inline src/components/Calculator.js]: - [x] Hi @karam084, Kindly fix Linters issues. We are not supposed to disable Linters. Linters are great tools that make sure that you are writing clean codes. They flag programming errors.

- I'll kindly advise you to extend React.PureComponent if you are struggling to resolve linters issues while using a class-base component, that's `class Calculator extends React.PureComponent {...`
- [ReadMore...](https://www.geeksforgeeks.org/reactjs-pure-components/)

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,12 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from 'react';
+import Calculator from './components/Calculator';
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (FrederickMih) [inline src/App.js]: - [x] Kindly check the comment above. Do not disable linters

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <button type="button" value="2" class="btn btn-light waves-effect">2</button>
+            <button type="button" value="3" class="btn btn-light waves-effect">3</button>
+
+
+            <button type="button" value="0" class="zerocal btn btn-light waves-effect ">0</button>
+            <button type="button" class="decimal function btn btn-secondary" value=".">.</button>
+
+          </div>
+        </div>
+      </div>
```
Comment (FrederickMih) [inline src/components/Calculator.js]: - [x] Great job on the fantastic design you came up with :+1: . Based on this project's requirements, kindly ensure that your design matches the screenshot below.


Screenshot from 2022-08-05 17-24-19

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @karam084 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights

- Correct use of GitHub workflow :heavy_check_mark: 
- Good PR title and summary :heavy_check_mark: 
- No Linter error :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi @karam084 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good PR title
✔️ Good PR description
✔️ Correct use of GitFlow
✔️ No linter errors

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**